step-framework-timeseries/src/main/java/step/core/timeseries/TimeSeries.java

```diff
@@ -46,6 +46,7 @@ public void ingestDataForEmptyCollections() {
                 logger.debug("Populating time-series collection: " + collectionName);
                 TimeSeriesCollection previousCollection = handledCollections.get(i - 1);
                 TimeSeriesIngestionPipelineSettings ingestionSettings = new TimeSeriesIngestionPipelineSettings()
+                        .setHandledAttributes(collection.getIgnoredAttributes())
```

Comment:
we're using two contradictory terms here, I guess setHandledAttributes should be changed for setIgnoredAttributes